src/keymap_manager.hh

```diff
@@ -45,13 +45,13 @@ public:
     };
     const KeymapInfo& get_mapping(Key key, KeymapMode mode) const;
 
-    using UserModeList = Vector<String>;
+    using UserModeList = HashMap<String, String, MemoryDomain::UserModes>;
```

Comment:
I would prefer to have a Vector<UserMode> with `struct UserMode { String name; String docstring; }`.

Reply:
That was what my original was using, but using a hashmap was so much easier… I'll change it back to a vector and rebase the PR.